File: Javascript/Exercicio02.js
const prompt = require('prompt-sync')();

var valores = new Array(4);
var soma = 0;
var multiplicacao = 1;

for (let i = 0; i < 4; i++) {
    valores[i] = prompt("Informe um valor" + (i + 1) + ": ");
}

valores.forEach(element => {
    element = parseFloat(element);
    soma += element;
    multiplicacao *= element;
});

console.log("a) Resultado de suas adições: "+soma);
console.log("b) Resultado de suas multiplicações: "+multiplicacao);

File: Java/Exercicio02.java
package Java;

import java.util.Scanner;

public class Exercicio02 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double soma= 0, multiplicacao = 1;

        double[] valores = new double[4];

        for (int i = 0; i < 4; i++) {
            System.out.print("Informe um valor" + (i + 1) + ": ");
            valores[i] = scanner.nextDouble();
        }
        for (double valor : valores) {
            //multiplicacao = valor * multiplicacao;
            multiplicacao *= valor;
            //soma = valor + soma;
            soma += valor;
        }

        System.out.print("a) Resultado de suas adições: "+soma+"\n");
        System.out.print("b) Resultado de suas multiplicações: "+multiplicacao);
        scanner.close();
    }
}
